spec/index.bs

```diff
@@ -4238,20 +4238,25 @@ dictionary GPUBlendComponent {
 enum GPUBlendFactor {
     "zero",
     "one",
-    "src-color",
-    "one-minus-src-color",
-    "src-alpha",
-    "one-minus-src-alpha",
-    "dst-color",
-    "one-minus-dst-color",
+    "src0-componentwise",
```

Comment:
`src0` seems confusing until we have a secondary color. Vulkan does `SRC` / `SRC1` and it seems pretty clear. I think we should keep as `src`.

Reply:
I think it's clearer, as there can still be multiple fragment outputs. Anecdotally, I never understood what "src" meant with multiple fragment outputs until someone told me that it always uses src0. That's not obvious behavior to me (especially when we don't have the OpenGL history of previously allowing only one fragment output).

Reply:
Have the same feelings as @Kangz . There is no need to treat `src1` as equal to `src`. Let's not make the general case more confusing than it needs to.

Reply:
I misunderstood how dual source blending worked. Changed src0 to src.